tests/Task.test.ts

```diff
@@ -77,6 +77,48 @@ describe('parsing', () => {
         expect(task!.tags[1]).toEqual('#journal/daily');
     });
 
+    it('parses a task from a line and extract tags and not global filter', () => {
+        // Arrange
+        const originalSettings = getSettings();
+        updateSettings({ globalFilter: '#task' });
+        const line =
+            '- [x] #task this is a done task #tagone #journal/daily 🗓 2021-09-12 ✅ 2021-06-20';
+        const path = 'this/is a path/to a/file.md';
+        const sectionStart = 1337;
+        const sectionIndex = 1209;
+        const precedingHeader = 'Eloquent Section';
+
+        // Act
+        const task = Task.fromLine({
+            line,
+            path,
+            sectionStart,
+            sectionIndex,
+            precedingHeader,
+        });
+
+        // Assert
+        expect(task).not.toBeNull();
+        expect(task!.description).toEqual(
```

Comment:
The test of description is useful and worth keeping.

Reply:
Covered as part of test refactor.